### ISSUE TITLE
Add Sky Correction to the Cookbook

### ISSUE THREAD

**Issue by heather999** [LSSTDESC/ImageProcessingPipelines#37]
There is a new feature which allows visit-level sky correction.  We briefly discussed this with Bob Armstrong at the 5 Apr 2018 SSim meeting and he recommended we look into using it for DC2.  Details are here:
https://community.lsst.org/t/hsc-defaults-now-require-sky-correction/2631

**Comment by heather999** [comment]
Looking over the community post, I think the first step is to determine how to build those sky frames, including what data to use to do so. Jim sounded interested in that too. Then we would need to demonstrate how well this sky correction works for DC2-era data.  Bob mentioned at the meeting that the sky correction cleaned up the sky nicely for the HSC data.  While we discussed it briefly at the SSim meeting, I didn't hear a clear ok from our group that we should definitely use sky correction for DC2 - we could still decide to keep it disabled for now.  So the final piece would be showing the results of enabling sky correction, maybe at a DC2 meeting or on the #desc-dc2 channel, and allowing others to comment.  Are you a willing volunteer @johannct ? :)

**Comment by johannct** [comment]
There will likely be several in the group looking into it. w_2018_14 is installed at CC I believe, so we can look into it.

**Comment by heather999** [comment]
Great - and just be aware that you'll ultimately need the LSSTDESC fork of obs_lsstSim after it has been rebased against the lsst master of obs_lsstSim.  Mostly that might be a reminder to myself :)

**Comment by heather999** [comment]
After talking with both @jchiang87 and @johannct last week, it seems we need the calibration products (biases, darks, and flats) before we can proceed. I was able to use phosim to generate biases (I think!). They are here: global/cscratch1/sd/desc/hmk/phosimTest/bias-output. I tried opening some files in ds9 and everything looks black to me :) but I'm hoping an intelligent user of ds9 could determine whether these files are useful or not.  
My first few runs to generate darks have timed out - initially I tried the 4 hour interactive queue, but that wasn't enough. I can try either a longer reservation, or perhaps there's a way to speed this up.  
Meanwhile, there has been discussion of using constructFlat.py on Slack https://lsstc.slack.com/archives/C77DDKZHR/p1523647664000397 We'll look at running it.

**Comment by heather999** [comment]
Actually I went back and took another look at the results of the bias images..  here's an example - might I have actually done this correctly? :)
![screen shot 2018-04-16 at 8 10 37 am](https://user-images.githubusercontent.com/10982136/38808397-09d4d320-414e-11e8-8ca7-743ba2e6d517.png)

**Comment by katrinheitmann** [comment]
Hi Heather,

Do you have the phosim inputs for this somewhere?

Thanks! Katrin

**Comment by jchiang87** [comment]
That visit time should be fine.  Let's discuss how to proceed on Slack.

**Comment by johannct** [comment]
cookbook descoped
